src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Shapes/EllipsePage.xaml

```diff
@@ -1,14 +1,43 @@
 ﻿<Page
-    x:Class="SamplesApp.Windows_UI_Xaml_Shapes.EllipsePage"
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    mc:Ignorable="d"
-    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+	x:Class="SamplesApp.Windows_UI_Xaml_Shapes.EllipsePage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	mc:Ignorable="d"
+	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
 	<StackPanel Spacing="5">
-		<TextBlock Text="This is a ellipse" />
-		<Ellipse x:Name="DEllipsePage"  Stroke="Black" Height="200" Width="250" StrokeThickness="5"/>
+		<TextBlock FontSize="22">This is an ellipse</TextBlock>
+		<Ellipse x:Name="DEllipsePage" Stroke="Black" Height="125" Width="150" StrokeThickness="5" HorizontalAlignment="Left"/>
+
+		<TextBlock FontSize="22">Those 3 ellipses should be identical (you should not see 4 ellipses!)</TextBlock>
+		<Grid>
+			<Grid.ColumnDefinitions>
+				<ColumnDefinition Width="175" />
+				<ColumnDefinition Width="20" />
+				<ColumnDefinition Width="Auto" />
+				<ColumnDefinition Width="20" />
+				<ColumnDefinition Width="175" />
+				<ColumnDefinition Width="20" />
+				<ColumnDefinition Width="*" />
+			</Grid.ColumnDefinitions>
+			<Grid Grid.Column="0" Width="175" Height="125">
+				<Rectangle StrokeDashArray="2.5,1" Stroke="Brown" StrokeThickness="3" />
+				<Ellipse Width="175" Height="125" Fill="DarkSalmon" x:Name="ellipse1" />
+			</Grid>
+			<Grid Grid.Column="2" Width="175" Height="125">
+				<Rectangle StrokeDashArray="2.5,1" Stroke="Brown" StrokeThickness="3" />
+				<Ellipse Fill="DarkSalmon" x:Name="ellipse2" />
+			</Grid>
+			<Grid Grid.Column="4">
+				<Rectangle StrokeDashArray="2.5,1" Stroke="Brown" StrokeThickness="3" />
+				<Ellipse Height="125" Fill="DarkSalmon" x:Name="ellipse3" />
+			</Grid>
+			<Grid Grid.Column="6" MaxWidth="175" HorizontalAlignment="Left">
+				<Rectangle StrokeDashArray="2.5,1" Stroke="Brown" StrokeThickness="3" />
+				<Ellipse Height="125" Fill="DarkSalmon" x:Name="ellipse4" />
+			</Grid>
+		</Grid>
 	</StackPanel>
```

Comment:
Can we also have a perfect circle example for Ellipse ;) ?

Reply:
Why?